<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnableTool</class>
 <widget class="QMainWindow" name="EnableTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>债券质押</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ICON.qrc">
    <normaloff>:/En/En.ico</normaloff>:/En/En.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>195</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>259</width>
          <height>197</height>
         </size>
        </property>
        <property name="title">
         <string>输入参数</string>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>231</width>
           <height>159</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="horizontalSpacing">
           <number>6</number>
          </property>
          <item row="2" column="1">
           <widget class="QComboBox" name="c_entrust_status">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="frame">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>已成</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>委托状态：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="c_entrust_price">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.980000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>业务品种：</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="c_entrust_direction">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>提交质押</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>转回质押</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>委托数量：</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="c_entrust_amount">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>1000000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="displayIntegerBase">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="c_stock_type">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <item>
             <property name="text">
              <string>债券</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>委托方向：</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>质押比例：</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>259</width>
          <height>196</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>259</width>
          <height>196</height>
         </size>
        </property>
        <property name="title">
         <string>功能函数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="sizeConstraint">
            <enum>QLayout::SetMaximumSize</enum>
           </property>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnExport">
             <property name="maximumSize">
              <size>
               <width>107</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>导出</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btnCalculate">
             <property name="maximumSize">
              <size>
               <width>107</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>计算</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnInit">
             <property name="maximumSize">
              <size>
               <width>107</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>系统初始化</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnClear">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>109</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetCurrent">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>123</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="autoScrollMargin">
         <number>20</number>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>26</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>21</number>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>初始状态</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>计算后状态</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>系统初始化后状态</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>持仓变化</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>业务品种</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>委托方向</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>委托数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>质押比例</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>委托状态</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>当前持仓数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>可用数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>冻结数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>解冻数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>质押数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>成本价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>成本</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>含费用成本</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>初始状态</string>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string>10000</string>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string>10000</string>
         </property>
        </item>
        <item row="0" column="8">
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item row="0" column="9">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="0" column="10">
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item row="0" column="11">
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item row="0" column="12">
         <property name="text">
          <string>1000000</string>
         </property>
        </item>
        <item row="0" column="13">
         <property name="text">
          <string>1000000</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>计算后状态</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>初始化后状态</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>138</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetStock">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>123</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>123</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>26</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>初始状态</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>计算后状态</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>系统初始化后状态</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>标准券变化</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>业务品种</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>委托方向</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>委托数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>质押比例</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>委托状态</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>当前持仓数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>可用数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>冻结数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>解冻数量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>成本价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>成本</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>含费用成本</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>初始状态</string>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="6">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="0" column="7">
         <property name="text">
          <string>10000</string>
         </property>
        </item>
        <item row="0" column="8">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="0" column="9">
         <property name="text">
          <string>10000</string>
         </property>
        </item>
        <item row="0" column="10">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="0" column="11">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="0" column="12">
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>计算后状态</string>
         </property>
        </item>
        <item row="2" column="0">
         <property name="text">
          <string>初始化后状态</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ICON.qrc"/>
 </resources>
 <connections/>
</ui>
